hydradx/tests/test_omnipool_amm.py

```diff
@@ -400,6 +400,302 @@ def test_swap_lrna(initial_state: oamm.OmnipoolState):
         raise AssertionError('Agent holdings are wrong.')
 
 
+@given(st.floats(min_value=10000, max_value=10000000),
+       st.floats(min_value=10000, max_value=10000000),
+       st.floats(min_value=10000, max_value=10000000),
+       st.floats(min_value=10000, max_value=10000000),
+       st.floats(min_value=10000, max_value=10000000),
+       st.floats(min_value=10000, max_value=10000000),
+       st.floats(min_value=0.0001, max_value=0.01),
+       st.floats(min_value=0.0001, max_value=0.01))
+def test_lrna_swap_buy_with_lrna_mint(
```

Comment:
It looks like we have added the same test functions to be test_omnipool_agents.py and test_omnipool_amm.py? I think these tests should be in test_omnipool_amm.py, right?

Reply:
I don't think these are in test_omnipool_agents - I can't find them there, even in a previous commit. Maybe the formatting on this page just made it look that way?

Reply:
You're right